lms/djangoapps/django_comment_client/management_utils.py

```diff
@@ -26,9 +26,11 @@ def get_mongo_connection_string():
                 user=MONGO_PARAMS['user'],
                 password=MONGO_PARAMS['password'],
             )
-    connection_string = "mongodb://" + credentials + "{host}:{port}/{database}".format(
-        host=MONGO_PARAMS['host'],
-        port=MONGO_PARAMS['port'],
+    host_ports = ["{host}:{port}".format(host=host_port['host'], port=host_port['port'])
```

Comment:
Let's clean up the syntax here...

```python
hosts = [
    "{host}:{port}".format(
        host=host['host'],
        port=host['port'],
    )
    for host in MONGO_PARAMS['hosts']
]
hosts = ','.join(hosts)
connection_string = "mongodb://{credentials}{hosts}/{database}".format(
    credentials=credentials,
    database=MONGO_PARAMS['database'],
    hosts=hosts,
)
```

A few notes:
- By convention, we use double-quotes for interpolated strings (like the call to `.format`) and single quotes most everywhere else.
- Avoid direct string concatenation, in favor of string interpolation/formatting. Definitely don't mix the two.